constants/Icons.js

```diff
@@ -0,0 +1,13 @@
+const EMOJIS = {
```

Comment:
Can we rename this file to something specific to restock indicators? Maybe RestockIcons.js? With just Icons.js it's possible this is just generalized icons.